src/main/java/org/recommendation/service/MovieRecommenderSystem.java

```diff
@@ -0,0 +1,108 @@
+package org.recommendation.service;
+
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.BufferedReader;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Scanner;
+
+import org.recommendation.data.DataInitializer;
+import org.recommendation.data.MovieDataInitializer;
+import org.recommendation.data.UserDataInitilizer;
+import org.recommendation.data.GenreDataInitializer;
+import org.recommendation.log.Logger;
+import org.recommendation.log.SoutLogger;
+
+public class MovieRecommenderSystem implements RecommenderSystem {
+    // using singleton design pattern
+    private static MovieRecommenderSystem recommenderSystem;
+
+    private MovieRecommenderSystem() {
```

Comment:
why this?

Reply:
Using Singleton Design Pattern, so making this constructor private so that we can't initialize this.